README.md

```diff
@@ -27,16 +27,10 @@ A third party security audit was performed by Cure53, you can see the full repor
 
 ## Building
 
-`runc` only supports Linux. It must be built with Go version 1.21 or higher.
+`runc` only supports Linux. See the header of [`go.mod`](./go mod) for the required Go version.
```

Comment:
This was changed. If we are going to have some text, the text before seem better. But I'm fine with this.

@kolyshkin what do you think? You weighted on this before.